src/emitters.ts

```diff
@@ -93,7 +93,7 @@ export const environmentChanged = new vscode.EventEmitter<EnvironmentChangeEvent
  * Cluster" action from the Topics view, or cleared out from a connection (or CCloud organization)
  * change.
  */
-export const currentKafkaClusterChanged = new vscode.EventEmitter<KafkaCluster | null>();
+export const topicsViewResourceChanged = new vscode.EventEmitter<KafkaCluster | null>();
```

Comment:
Future branch: do we want to also rename `currentSchemaRegistryChanged` to `schemasViewResourceChanged`?